src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0 row">
```

Comment:
`stars--0` should be a bem modifier of `stars` block. so you should create this block
the same with `stars__star`, it's a bem element. these can't exist without a bem block.